src/sentry/static/sentry/app/components/group/suggestedOwners/styles.tsx

```diff
@@ -0,0 +1,21 @@
+import styled from '@emotion/styled';
+
+import space from 'app/styles/space';
+
+const Heading = styled('div')`
```

Comment:
Should we use an actual heading element here?

Reply:
you are right, headings are important, but I believe that we should review the spaces that accompany our h* elements ... otherwise, I have to use` !important` everywhere .. and this is not good practice. 

I have updated the code 😉

Reply:
Would having a module of styled components that provides some headings with the margins stripped off help? I agree that having to apply important all over the place is not great.